packages/server/routes/api/guild.js

```diff
@@ -4,30 +4,55 @@ const router = express.Router();
 const GuildController = require("../../controllers/guild");
 const AuthController = require("../../controllers/auth");
 
-router.get('/', async (request, response) => {
+router.get("/", async (request, response) => {
   try {
     const guilds = await GuildController.getAllGuild();
     response.send(guilds);
-  } catch(error) {
+  } catch (error) {
     response.send({
       success: false,
-      message: error.message
+      message: error.message,
     });
   }
 });
 
-router.get('/:guildId', AuthController.authenticate, async (request, response) => {
-  try {
-    console.log("@user", request.user);
-    const { guildId } = request.params;
-    const guild = await GuildController.getEvent(guildId);
-    response.send(guild);
-  } catch(error) {
-    response.status(403).send({
-      message: error.message,
-      success: false
-    });
+router.post(
+  "/insert",
+  // AuthController.authenticate,
+  async (request, response) => {
+    try {
+      const newGuild = await GuildController.insertGuild(request.body);
```

Comment:
When inserting, some of the names are different for the guild properties like "title" vs "name", "bannerUrl" vs "banner", "isInviteOnly" vs "invite_only" (at least when I was testing using Postman). I believe they should be the same all the time.